### ISSUE TITLE
CTD when moving to a location with "captive" faction

### ISSUE THREAD

**Issue by NewOrlanth** [CleverRaven/Cataclysm-DDA#29600]
**Describe the bug**  
I ran into a military outpost with bandits. Killed them and moved forward. A few tiles later, CTD with this message :
ERROR : src/faction.cpp:310 [faction* faction_manager::get(const faction_id&)] Requested non-existing faction 'captives' 

**To Reproduce**  
I don't know how to reproduce it elsewhere. First time i met captives.

**Expected behavior**  
Not CTD :)

**Versions and configuration(please complete the following information):**
 - OS: Windows 10
 - Game Version: ..D-1799-g43f1f6e
 - Graphics version: Tiles
 - Mods loaded: [ "dda",  "no_npc_food",  "filthy_morale",  "ew_pack",  "Medieval_Stuff",  nw_pack",  "alt_map_key",  "blazemod",  "Tanks",  "aftershock",  "FIC_Weapons",  "MMA",  "more_locations",  "cityside",  "FujiStruct",  "deoxymod"]

**Additional context**  
Link to savegame : http://jif.memoiresecondaire.fr/f.php?h=1ZP_ptKe&d=1
Just go a few tiles east.

**Comment by Little-119** [comment]
I think the captive faction is added by More_Locations. Have you modified that mod's files in any way?

**Comment by NewOrlanth** [comment]
Captive is in more_location but i didn't change anything :/

**Comment by NewOrlanth** [comment]
Following the idea, i just downloaded brand new latest Cdda version, put my save in it without touching anything else and ran it. 
Unfortunately no change, still CTD.

**Comment by NewOrlanth** [comment]
I may add that it's the same with refugee. Probably a problem with more-locations that i can't correct myself. 
Seems to add correctly building and everything but no factions.

**Comment by davidpwbrown** [comment]
The faction code is changing, probably that mod needs to change the way it does the faction. @mlangsdorf  would know more about the faction code changes currently beign worked on.

**Comment by NewOrlanth** [comment]
I think it was my fault. 
After some research i found that factions definitions are saved in master.gsav
I created another character in another world, with more_location activated, and checked factions in master.gsav and found refugee and captive correctly put. 
I copied the file in my regular save, and now it doesn't CTD anymore. 

I probably activated more_location after initial character and world creation without remembering.
Anyway, problem on me. 
Thanks anyone, this issue can be closed :)
